README.md

```diff
@@ -1,40 +1,41 @@
 
-![simple action pipeline logo](assets/img/SAP-small.png)
+![simple action pipeline logo](docs/img/SAP-small.png)
 ## simple-action-pipeline (SAP)
 
-A simple pipeline framework for implementing scheduled sequential and parallel actions. Designed around two simple configuration files and implementation of an uncomplicated workflow manager. The simple action pipeline uses the python package "[**Jug**](https://jug.readthedocs.io/en/latest/)" as it's underlying workflow manager. 
+A simple pipeline framework for implementing sequential, parallel and dependant actions. Designed around two yaml configuration files and implementing an uncomplicated workflow manager. The simple action pipeline uses the python package "[**Jug**](https://jug.readthedocs.io/en/latest/)" as it's underlying workflow manager and is provided as a pip installable python3 package. 
 
 The simple action pipeline has some standard features which wrap around any application specific configuration or code.  
 
-![simple action pipeline features](assets/img/pipeline-features.png)
+![simple action pipeline features](docs/img/pipeline-features.png)  
+
+Full documentation for *simple-action-pipeline* can be found on [Github Pages](https://antarctica.github.io/simple-action-pipeline/).
 
 ##  
 
-### Installation
-It is recommended that the sap package is installed and used within a Python virtual environment. The Python version must be >= 3.9.x .  
+## Installation
 
-To create a Python virtual environment:  
-```
-python -m venv path_to_new_virtual_env
-```
-  
-Then activate the python virtual environment:  
-```
-source path_to_new_virtual_env/bin/activate
-```  
+It is recommended to use a Python virtual environment to reduce the risk of any Python package conflicts.
 
-Once activated, install the sap package and it's dependencies into the python virtual environment:  
-```
-git clone https://github.com/antarctica/simple-action-pipeline ./simple-action-pipeline
-cd ./simple-action-pipeline
-pip install -e .
-```  
+### HPC Workstation or Local PC
```

Comment:
```suggestion
```

I wouldn't bother saying whether it's for an HPC workstation or local PC since the install instructions are straightforward enough to work for any ordinary python envrionment.